pages/packages/maven.md

```diff
@@ -48,18 +53,85 @@ The following steps describe the process above:
     // distributionManagement: 'maven deploy' pushes to this repository
     <distributionManagement>
       <repository>
-        <id>org-slug-java-maven-package-repository-name</id>
-        <url>https://buildkitepackages.com/{org.slug}/{java.maven.package.repository.name}</url>
+        <id>org-slug-java-package-repository-name</id>
+        <url>https://buildkitepackages.com/{org.slug}/{java.package.repository.name}</url>
       </repository>
       <snapshotRepository>
-        <id>org-slug-java-maven-package-repository-name</id>
-        <url>https://buildkitepackages.com/{org.slug}/{java.maven.package.repository.name}</url>
+        <id>org-slug-java-package-repository-name</id>
+        <url>https://buildkitepackages.com/{org.slug}/{java.package.repository.name}</url>
       </snapshotRepository>
     </distributionManagement>
     ```
 
+    where:
+    * `org-slug-java-package-repository-name` is the ID of your Java package repository (above).
+
+    <%= render_markdown partial: 'packages/org_slug' %>
+
+    <%= render_markdown partial: 'packages/java_package_repository_name' %>
+
 1. Publish your package:
 
     ```bash
     mvn deploy
     ```
+
+## Access a package's details
+
+<%= render_markdown partial: 'packages/access_java_package_details_page' %>
+
+<%= render_markdown partial: 'packages/package_details_page_sections' %>
+
+### Downloading a package
+
+A Java (Jar) package can be downloaded from the package's details page.
```

Comment:
```suggestion
A Java package can be downloaded from the package's details page.
```

Not all Java packages are Jars.

Reply:
Thanks! Removed other references to these throughout these docs.